.github/workflows/dart.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [stable, beta]
```

Comment:
My preference is to convert this to an expanded list and then comment out the beta line with a reference to the trackiing bug, akin to how we handle CI on `flutter/samples`

Reply:
Makes sense. I've added a link to the issue and switched it so `beta` is marked as experimental, so we can still reference the build results as needed.